submissions/m-ruslan/friends-app/src/sorter.js

```diff
@@ -0,0 +1,147 @@
+let users = [];
+let sorted = [];
+let filtered = [];
+let nameSearched = [];
+let ageSearched = [];
+const state = {
+  sortBy: "",
+  sortOrder: "",
+  filterGenderBy: "",
+  searchByName: "",
+  ageFrom: "",
+  ageTo: "",
+};
+
+let drawResult = null;
+
+const getDatasetValue = (elements, key) => {
+  const checked = Array.from(elements).find(({ checked }) => checked);
```

Comment:
You can replace this with a query selector.

And when you will do, you probably will understand that there is no need for a separate function.